common-tasks/working-with-office/en.md

```diff
@@ -0,0 +1,8 @@
++++
+title = "Working with office"
++++
+# Working with office
+
+Solus comes pre-installed with LibreOffice, an office suite that is capable of replacing Microsoft Office for the most common tasks.
+Unless you are a heavy power user, LibreOffice should provide you with all the tools needed to solve your daily office challenges.
```

Comment:
There should be a newline before this, otherwise it isn't going to render correctly.

> Unless you are a heavy power user,

Unless we're trying to classify what is and isn't a power user, this shouldn't be here. It also hasn't been defined **what** LibreOffice may not be capable of doing, thus even for "power users", it is quite possible that LibreOffice is entirely sufficient.

> LibreOffice should provide you with all the tools needed to solve your daily office challenges.

Such as? We have pre-installed **some** of LibreOffice, those applications should be indicated and instructions should be provided on installing other LibreOffice programs like LibreOffice Math.